src/routers/api.py

```diff
@@ -11,12 +14,21 @@ def test():
     return "petition"
 
 
+
 router.include_router(
     petitions,
     prefix="/petitions",
     tags=["petitions"],
     responses={404: {"description": "Not found"}},
 )
+
+router.include_router(
+    auth,
+    prefix="/auth",
+    tags=["api", "account", "auth"],
```

Comment:
```suggestion
    tags=["auth"],
```
여러가지 태그를 동시에 적으면, swagger나 redoc에서 같은 api가 중복으로 떠요. path에 맞게 auth만 사용하는게 더 나아보여요